### DISCUSSION TITLE
Pi 4 versus older ones

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#711] [Q&A]
I have (literally) 20 -30 rpis laying around from 2, through 3a+, 3b, and 3b+.

I use the older ones in irrigation controller boxes and various home automation applications.  I consider these applications demand very little CPU and memory resource.

So, why specifically the RPI4 recommendation versus just an RPI?   

Thanks!

**Comment by rstrouse** [comment]
Part of that is simply because at the time of the writing there was no reason to go out and buy anything else.  That being said folks often install mosquitto, Influx, and Grafana on that same pi.  By the time you do that, a pi 3 is pretty much at its limits.  There are a few that are running Pi Zeros but I wouldn't recommend that.  Those cost as much as a 4 these days.

**Reply by davthomaspilot** [reply]
Yeah, I hear you.

But nowadays, there's a big difference between buying anything new versus having them on hand.  Besides, this is for a chem controller only.  If I someday replace the Intellitouch panel, it will be at the other side of the equipment area and will use another pi.

Thanks!
